### ISSUE TITLE
GHC 9.4 compatible release

### ISSUE THREAD

**Issue by locallycompact** [haskell-beam/beam#649]
Hi can we get a hackage release for 9.4 please?

**Comment by peterbecich** [comment]
There may be one more dependency to resolve, `hint`. With GHC 9.4.2:
```
$ cabal build all --enable-tests

Error: cabal: Could not resolve dependencies:
[__0] trying: beam-core-0.10.0.0 (user goal)
[__1] trying: base-4.17.0.0/installed-4.17.0.0 (dependency of beam-core)
[__2] trying: beam-migrate-cli-0.2.1.0 (user goal)
[__3] trying: hint-0.9.0.6 (dependency of beam-migrate-cli)
[__4] next goal: ghc (dependency of hint)
[__4] rejecting: ghc-9.4.2/installed-9.4.2 (conflict: hint => ghc>=8.4 &&
<9.3)
...
```

**Comment by peterbecich** [comment]
GHC 9.4 support for `hint`:
https://github.com/haskell-hint/hint/issues/151
https://github.com/haskell-hint/hint/pull/152

**Comment by tathougies** [comment]
I will look into upgrading the dependencies and making a hackage release. `beam-migrate-cli` is the only component dependent on hint I believe.

**Comment by tathougies** [comment]
As of last week, I believe all packages on hackage have been updated, except for beam-migrate-cli, which should not be used at this point.

**Comment by tathougies** [comment]
https://hackage.haskell.org/package/beam-core-0.10.0.0
https://hackage.haskell.org/package/beam-postgres-0.5.3.0
https://hackage.haskell.org/package/beam-sqlite-0.5.2.0

**Comment by gelisam** [comment]
> There may be one more dependency to resolve, `hint`

[hint-0.9.0.7](https://hackage.haskell.org/package/hint-0.9.0.7) now supports ghc-9.4 and ghc-9.6

**Comment by kmicklas** [comment]
Fixed by #691.
